vignettes/implementation.Rmd

```diff
@@ -104,7 +104,9 @@ nd_grts(list(x = x), new.length = 2 ^ n)
 
 In some cases we want a fixed scaling between some of the variables. An obvious example is a spatial layout were it makes more sense to keep a fixed scaling between both coordinates so that the distance between the points remains Euclidean. Therefore we added two more arguments to `nd_grts()`: `reference` and `scale`. `reference` must be a single character with the name of the dimension which is used as reference. `scale` is a named numeric vector defining the scale. Scale is somewhat analogue to the scale on maps: one unit of the reference equals $s$ units of the scaled variable. E.g. `reference = "A"` and `scale = c(B = 2, C = 1)` implies that 1 unit of "A" is as large as 2 units of "B" and as 1 unit of "C". 
```

Comment:
The piece on 'fixed scaling' somehow breaks the flow. In the previous part, GRTS was well explained for one and for n dimensions. Here more clarity on the reasons and implications of scaling for the GRTS algorithm would be helpful, i.e. in the context of the previous part. Earlier, it was stated that each vector (dimension) is split in intervals so that each interval contains one element at most. This is broken here, e.g. by taking a specific cellsize. This can be seen in the SpatialPoints example, where I added a little explanation.